cassandra-mesos-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/scheduler/CassandraCluster.java

```diff
@@ -448,6 +451,18 @@ private CassandraNode buildCassandraNode(@NotNull final Protos.Offer offer, fina
         }
     }
 
+    @NotNull
+    private RackDc getRackDc(@NotNull final Protos.Offer offer) {
+        final String rack = attributeValue(offer, RACK_ATTRIBUTE);
```

Comment:
Could you add a descriptive comment here? Also, the implementation is unchanged if the agent node attributes are not set, correct?